gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/service/MinioHandler.java

```diff
@@ -258,52 +243,48 @@ private String getFileEventMessage(FileEventType fileEventType, String fileType,
     }
 
     public void emptyWaitingList(OffsetDateTime timestamp) {
-        List<ProcessFileMinio> processFileToProcess = getProcessFileMinios(timestamp);
-        List<TaskWithStatusUpdate> setTaskToNotify = new ArrayList<>();
-        boolean processEventAdded = false;
-        for (ProcessFileMinio processFileMinio : processFileToProcess) {
-            List<OffsetDateTime> listTimestamps = Stream.iterate(processFileMinio.getProcessFile().getStartingAvailabilityDate(), time -> time.plusHours(1))
-                    .limit(ChronoUnit.HOURS.between(processFileMinio.getProcessFile().getStartingAvailabilityDate(), processFileMinio.getProcessFile().getEndingAvailabilityDate())).collect(Collectors.toList());
-
-            List<TaskWithStatusUpdate> listTaskWithStatusUpdate = findAllTaskByTimestamp(listTimestamps);
-            if (atLeastOneTaskIsRunningOrPending(listTaskWithStatusUpdate)) {
-                continue;
-            }
-            for (TaskWithStatusUpdate taskWithStatusUpdate : listTaskWithStatusUpdate) {
-                Task task = taskWithStatusUpdate.getTask();
-                checkAndUpdateTaskStatus(task);
-                if (!processEventAdded && task.getStatus().equals(TaskStatus.READY)) {
-                    task.addProcessEvent(getProcessNow(), "WARN", "Task has been set to ready again because new inputs have been uploaded. Output files might be outdated.");
-                    processEventAdded = true;
-                }
+        LOGGER.info("Empty waiting list..");
```

Comment:
Wording: when I first read this log, I thought you meant that the waiting was empty, but then just at the following line you write in your log the number of elements contained in the list...
To make it clearer, maybe change "empty" to "handle / handling"?